website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+
+There are two ways to showcase an upgradable feature:
+
+- Space to explain
+- Interrupting an action
+
+### Space to explain
+
+In this example, "Terraform Health" is a an upgradable feature and is contextually relevant in the workspace. The UI has space to entice the user by explaining a bit of what this service can do for them.
+
+![Screen shot of Terraform workspace highlighting Terraform Health on the side bar below information.](/assets/patterns/disabled-patterns/upgrade-inline-example.png)
+
+### Interrupting an action
+
+In this example, a user is about to perform an action and notices something new within the list of options, one with a "Beta" badge. 
+
+![Showing a dropdown opened, with two list items. One for creating a static secret, while another using an auto-rotating secret with a beta badge next to it.](/assets/patterns/disabled-patterns/upgrade-modal-example-1.png)
+
+Enticed, they click it to use this feature, however, they are not enrolled in this tier and are interrupted by a modal explaining this feature a little more.
+
+![A modal explaining beta to the user, with a CTA to learn more, or close the modal.](/assets/patterns/disabled-patterns/upgrade-modal-example-2.png)
+
+Since the action is the only explanation a user has at the moment, providing a modal as an interruption to the action allows a user to understand more without fully committing. Only when a user wants to "learn more," they can then be lead to a page with more details.
+
+
+## Service outage
+
+In instances where outages may occur, provide a clear message to users on what they can do and add a link to [our status page](https://status.hashicorp.com/).
+
+![Displaying Hashicorp's 500 internal error, providing a retry button and a link to the the status page.](/assets/patterns/disabled-patterns/service-outage-example.png)
+
+## Incomplete flow
+
+In a form, always enable the submit button. When buttons are disabled, users are often left confused and frustrated because they can't move forward. Instead, show users what errors occurred so they can fix them and proceed. 
+
+!!! Do
+
+When a user clicks "Create cluster" but has failed to enter a value into a `required` input, an error will display, informing the user how to fix the issue.
+
+![Image showing an input field with an error explaining what was missed in a form context.](/assets/patterns/disabled-patterns/incomplete-flow-example.png)
+
+!!!
+
+Learn more about [form validation patterns](/patterns/form-patterns?tab=validation).
+
+## Quota limitation
+
+When a quota has been reached, hide the action allowing them to create more objects. If a user can increase the quota, replace the original action with an action that allows them to do so. If the user cannot increase the quota, e.g., they don't have permission, direct them to a form.
```

Comment:
[question] when you say "direct them to a form" what do you mean here? Do you mean direct them to a form where they can change permissions?

Reply:
This is coming straight from HCP's guidelines, and it was intentionally left open ended because it could be to change permissions, contact an admin, or really anything in between (could also be to input payment details). Initially I changed it to "contact an admin," but was reverted back because that wasn't in the initial guidelines.

Reply:
Ok got it, thanks for the clarification there!